library/general/src/modules/Linuxrc.rb

```diff
@@ -211,6 +211,31 @@ def SaveInstallInf(root)
       true
     end
 
+    # Returns value of a given Linxurc key/feature defined on commandline
+    # and written into install.inf
+    #
+    # @param [String] key
+    # @return [String] value of a given key
+    def value_for(feature_key)
+      ReadInstallInf()
+
+      feature_key = polish(feature_key)
+      install_inf_features = deep_copy(@install_inf)
+
+      # Some values written on Linuxrc commandline are not known to Linuxrc
+      # Unless they are also mentioned as `PTOptions`, they will appear as "Cmdline" entry
+      install_inf_features.fetch("Cmdline", "").split.map do |cmdline_entry|
+        key_val = cmdline_entry.split("=")
+        install_inf_features[key_val[0]] = key_val[1]
+      end
+
+      matching_keys = install_inf_features.keys.select { |key| polish(key) == feature_key }
```

Comment:
Another issue is that order of `install_inf_features.keys` is not guaranteed, currently `Hash#keys` returns the keys in the insert order (http://stackoverflow.com/questions/5208512/ruby-what-is-the-order-of-keys-values-returned-by-hash-keys-and-hash-values-met) but it was different with Ruby 1.8 and might change anytime in the future....